<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::SkyDisplayDialog</class>
 <widget class="QDialog" name="snowgui::SkyDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="snowgui::SkyDisplayWidget" name="skydisplayWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>512</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="skyoptionWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="radecCheckBox">
        <property name="text">
         <string>RA/DEC grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="azmaltCheckBox">
        <property name="text">
         <string>Azimuth/Altitude grid</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="eclipticCheckBox">
        <property name="text">
         <string>Ecliptic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="constellationsCheckBox">
        <property name="text">
         <string>Constellations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="telescopeCheckBox">
        <property name="text">
         <string>Telescope position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="targetCheckBox">
        <property name="text">
         <string>Show target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="labelsCheckBox">
        <property name="text">
         <string>Show direction labels</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::SkyDisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">SkyDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
